deps/rabbitmq_amqp1_0/README.md

```diff
@@ -51,6 +51,14 @@ Configuration example using [sysctl config format](https://next.rabbitmq.com/con
     amqp1_0.protocol_strict_mode = false
 
 
+Configuration of incoming links: 
+
+```
+amqp1_0.maximum_incoming_credit = 65536
+# Should be set at on_confirm for stability of RabbitMQ
+amqp1_0.grant_credit = on_publish | on_confirm (default on_publish for backwards compatibility)
```

Comment:
These names of these settings assume the use of AMQP legacy (amqp_client) internally which is something we will move away from, hopefully sooner rather than later. That said it is probably ok for now as I very much doubt we would ever grant credit on_publish in the future and we can just deprecate this config setting.